src/platforms/macos/macosutils.mm

```diff
@@ -89,3 +89,42 @@ bool dockClickHandler(id self, SEL cmd, ...) {
 void MacOSUtils::showDockIcon() {
   [NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
 }
+
+void MacOSUtils::installerStrings() {
+  //% "Mozilla VPN for macOS"
+  qtTrId("macosinstaller.title");
+
+  //% "Unable to install"
+  qtTrId("macosinstaller.unsupported_version.title");
+
+  //% "Mozilla VPN requires Mac OS X 10.6 or later."
+  qtTrId("macosinstaller.unsupported_version.message");
+
+  //% "Previous Installation Detected"
+  qtTrId("macosinstaller.previous_build.title");
+
+  //% "A previous installation of Mozilla VPN exists at /Applications/Mozilla VPN.app. This"
```

Comment:
You need a space at the end of these sentence that continue on a new line. 

This is how the string currently looks like when exported

```
    <message id="macosinstaller.previous_build.message">
        <source>A previous installation of Mozilla VPN exists at /Applications/Mozilla VPN.app. Thisinstaller will remove the previous installation prior to installing. Please back up any databefore proceeding.</source>
        <translation type="unfinished"></translation>
    </message>
```